.github/workflows/windows.yml

```diff
@@ -62,6 +62,7 @@ jobs:
 
 
   package-exe:
+    if: ${{ github.event_name == 'push' }}
```

Comment:
For what do we need those guards?

Reply:
In the PR there's no access to secrets, therefore upload fails

Reply:
Though it may be a good idea to move the guards closer to upload itself. Then it'll be possible to download the build artifact from the PR even without creating the release